src/components/Calendar/EventListSideBar.js

```diff
@@ -29,13 +29,16 @@ function EventListSideBar({ showModal, setClickSelected, setSelectLocation, even
     />
   );
 
+  const [selectedKey, setSelectedKey] = useState('3')
+
   //change handler for select location
   const handleLocationChange = (e) => {
     setSelectLocation(e.item.props.value)
+    setSelectedKey(e.item.props.eventKey)
   }
 
   const schoolMenu = (
-    <Menu onClick={handleLocationChange}>
+    <Menu onClick={handleLocationChange} selectedKeys={selectedKey}>
       <Menu.Item value="India" key="1">India</Menu.Item>
       <Menu.Item value="Africa" key="2">Africa</Menu.Item>
       <Menu.Item value="" key="3">All Locations</Menu.Item>
```

Comment:
Great idea adding the key values to show what location is currently selected. Nice work Pete. Thank you so much for being a great team player. Hopefully, we will no longer see any merge conflicts!